changelogs/CHANGELOG.rst

```diff
@@ -4,6 +4,28 @@ Cisco Asa Collection Release Notes
 
 .. contents:: Topics
 
+v1.0.2
+======
+
+Release Summary
+---------------
+
+- Re-releasing 1.0.2 with updated changelog.
+
+v1.0.1
+======
+
+Minor Changes
+-------------
+
+- Add changelogs for 1.0.0 release (https://github.com/ansible-collections/cisco.asa/pull/48)
```

Comment:
The first entry is not required. This got generated because the changelog PR for 1.0.0 had a changelog entry.